### PR TITLE
Flatten nested data so table does not crash

### PR DESCRIPTION
**Pull request by NathanielScottStevens**
smartcitiesdata/react_discovery_ui#55

co-authored-by: ManApart <akucera@pillartechnology.com>

### CODE REVIEW

File: src/pages/query-view/query-view.js [hunk 1]
```diff
@@ -86,8 +86,13 @@ const cleanseRow = row => {
 const cleanseField = value => {
   if (typeof value === "boolean") {
     return value.toString();
+  } else if (typeof value === "object") {
+    return JSON.stringify(value);
+  } else if (value === undefined) {
+    return ''
```
Comment (mcsearchin) [inline src/pages/query-view/query-view.js]: Might want to check for `null` and `NaN` in addition to `undefined`.

File: src/pages/query-view/query-view.js [hunk 2]
```diff
@@ -86,8 +86,13 @@ const cleanseRow = row => {
 const cleanseField = value => {
   if (typeof value === "boolean") {
     return value.toString();
+  } else if (typeof value === "object") {
```
Comment (mcsearchin) [inline src/pages/query-view/query-view.js]: I think this will work for arrays too, but you may want to double check.

**Comment by mcsearchin** [comment]
Did the bug in the data dictionary table turn out to be bigger than we wanted to deal with in this PR?

**Comment by ManApart** [comment]
> Did the bug in the data dictionary table turn out to be bigger than we wanted to deal with in this PR?

We're going to ask Carrie about it this morning for 'entry'.

File: src/pages/query-view/query-view.js (hunk 1 above)
Comment (NathanielScottStevens) [reply src/pages/query-view/query-view.js]: We learned that undefined is unnecessary but added null and NaN.

File: src/pages/query-view/query-view.js (hunk 2 above)
Comment (NathanielScottStevens) [reply src/pages/query-view/query-view.js]: It does! Thanks for thinking of that, didn't even occur to us to be afraid of arrays.

File: src/pages/query-view/query-view.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import React from "react";
 import DatasetQuery from "../../components/dataset-query";
 import ReactTable from "react-table";
 import LoadingElement from "../../components/generic-elements/loading-element";
+import _ from 'lodash'
```
Comment (bbrewer-papa-x) [inline src/pages/query-view/query-view.js]: Alternatively, if you don't want to have to carry the `_` around, I like to just import the functions we need

Comment (NathanielScottStevens) [reply src/pages/query-view/query-view.js]: We're importing isNull and isNaN so I figured it was better to call out that they were not the builtins.

**Pull request merged**